### DISCUSSION TITLE
[@types/async-busboy] Wrong type in v0.7.2

### DISCUSSION THREAD

**Discussion by kerberjg** [DefinitelyTyped/DefinitelyTyped#56786] [Issues with a @types package]
In @types/async-busboy v0.7.2, the `onFile()`'s `file` parameter has the wrong type: instead of being `Readable` it's `NodeJS.ReadableStream`.

Here's the link to the offending types line: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6bc29be9be89024bd6dcce8c01e7725fd178cb66/types/async-busboy/index.d.ts#L16

Here are the links supporting the change:
- https://github.com/m4nuC/async-busboy/blob/3da5e7eb9021e3180f9fd03bbb8bc1b988eed31c/index.js#L25
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6bc29be9be89024bd6dcce8c01e7725fd178cb66/types/busboy/index.d.ts#L128
- https://github.com/mscdex/busboy/blob/5ef3f9b1d08e7ccb7c9b3690e4c071089e66285c/lib/types/multipart.js#L8

[a comment by an automated account is not shown]

**Comment by hirochachacha** [comment]
I don't remember about this code well, but I guess we can refer https://github.com/DefinitelyTyped/DefinitelyTyped/blob/56b2a8f3254e2c74470b9feadd8f2f2e669fea30/types/busboy/index.d.ts#L126-L132 directly instead of defining the type by ourselves.
If you're going to make PR, I'll review it.

**Reply by BendingBender** [reply]
Made a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56803
